kolibri/utils/conf.py

```diff
@@ -54,6 +54,9 @@
     "kolibri_exercise_perseus_plugin"
 ]
 
+#: Everything in this list is added to django.conf.settings.INSTALLED_APPS
+config['INSTALLED_APPS'] = DEFAULT_PLUGINS
```

Comment:
@rtibbles hmm, okay, would you agree that the logic sentiment here is that *unless we add version information stating when a default plugin was added as a default, it will be impossible to guarantee that plugins are automatically switched on if we decide to let local configuration of enabled plugins take precedence*?